flink-filesystems/flink-s3-fs-base/src/main/java/org/apache/flink/fs/s3/common/FlinkS3FileSystem.java

```diff
@@ -81,7 +81,7 @@ public class FlinkS3FileSystem extends HadoopFileSystem
         implements EntropyInjectingFileSystem, PathsCopyingFileSystem {
     private static final Logger LOG = LoggerFactory.getLogger(FlinkS3FileSystem.class);
 
-    private static final long PROCESS_KILL_SLEEP_TIME_MS = 50L;
+    private static final long PROCESS_KILL_SLEEP_TIME_MS = 1000L;
```

Comment:
I see [https://flink.apache.org/how-to-contribute/contribute-code/](https://flink.apache.org/how-to-contribute/contribute-code/) says hotfix is "Note: trivial hot fixes such as typos or syntax errors can be opened as a [hotfix] pull request, without a Jira ticket."

I do not think this is that trivial. I suggest raising a Jira to indicate what problem you are looking to fix. 

Please could you add a comment to say why you are increasing this sleep time by 20 times.